onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -1063,6 +1076,11 @@ def _where_input_wrangler(
         matcher=lambda sample: not (len(sample.args) > 0 and isinstance(sample.args[0], int)),
         reason="this Aten overload only support one tensor as input and one int as args by design",
     ),
+    skip(
+        "unflatten",
+        matcher=lambda sample: any(dim == 0 for dim in sample.input.shape),
+        reason="0 dim in ONNX is undefined behavior.",
+    ),
```

Comment:
Can we create a condition to handle this case?

Reply:
To raise a warning? Not sure what we should give back to users.

Reply:
What is pytorch's behavior?

Reply:
PyTorch takes this because 0-d is in their spec.
For example,
```python
a = torch.randn(4, 0)
# tensor([], size=(4, 0))

torch.unflatten(a, 0, (2, 2))
# tensor([], size=(2, 2, 0))
```

Reply:
I wonder if torch_libs has a certain way to inform users when the difference between specs is hit?

Reply:
I see. We could skip this if it is too complicated and come back to it as need. I would update the skip reason to say something to the effect of "fixme: logic not implemented for size 0 inputs"

Reply:
We don't raise errors in atenlib. The exporter can raise errors when it sees a difference after OpSchema is implemented